corehq/apps/cloudcare/static/cloudcare/js/formplayer/router.js

```diff
@@ -151,6 +151,12 @@ hqDefine("cloudcare/js/formplayer/router", function () {
             urlObject.addSelection(index);
             urlObject.setForceManualAction(false);
         }
+        if (smartLinkParams && smartLinkParams.length) {
+            var userDomain = FormplayerFrontend.getChannel().request('currentUser').domain;
+            if (smartLinkParams[0] !== userDomain) {
```

Comment:
is it necessary to use a generic name like `smartLinkParams` if we're always going to be comparing this to the domain? would it be better to just call it `smartLinkDomain`?

I'm also wondering how well this translates to mobile (if at all).

Reply:
I'm ambivalent about this, because I'd like formplayer to not know anything about domains (even though formplayer does reference `commcare_project`). I made some adjustments to this so it's still a "params" list but it's now named params.